client/me/help/help-results/style.scss

```diff
@@ -13,33 +13,22 @@
 
 .help-result {
 	&:hover {
-		background-color: $blue-medium;
-		.help-result__title,
-		.help-result__description {
-			color: $white;
+		.help-result__title {
+			color: $blue-medium;
 		}
-		.help-result__icon-path {
-			fill: $white;
-		}
-	}
-}
-
-.help-result__wrapper {
-	&:hover {
-		background-color: $blue-medium;
 	}
 }
 
 .help-result__icon {
 	position: absolute;
 	margin: -2px 16px 0px -8px;
+	fill: $blue-wordpress;
```

Comment:
@roundhill do you mind changing this to $gray? I noticed that I accidentally changed it to blue. I would do it, but I think I screwed up my local copy.